api/src/controllers/put/mine.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const SYSTEM_USER = 'SYSTEM_USER';
+
+/**
+ * Performs all operations necessary to edit a master Mine record.
+ *
+ * @param {*} args
+ * @param {*} res
+ * @param {*} next
+ * @param {*} incomingObj see example
+ * @returns object containing the operation's status and created records
+ */
+exports.editRecord = async function(args, res, next, incomingObj) {
+  try {
+    if (incomingObj._schemaName !== 'MineBCMI') {
+      throw new Error('editRecord - incorrect schema type, must be MineBCMI');
+    }
+
+    const Model = mongoose.model('MineBCMI');
```

Comment:
Similar comment to the mine Post constoller: 

For master records, the model/schema is Mine.

The flavour record will be MineBCMI

Reply:
Updated